generators/gulp/templates/gulpfile.js

```diff
@@ -42,7 +42,8 @@ gulp.task('test', ['pre-test'], function (cb) {
 
   gulp.src('test/**/*.js')
     .pipe(plumber())
-    .pipe(mocha({reporter: 'spec'}))
+    .pipe(mocha({reporter: 'spec'<% if (babel) { %>,
+      require: ['babel/register']<% } %>}))
```

Comment:
Yeah, just muck with it. mine just an idea what you could do (take advantage with the new templating system and the styling of the object).